src/deltacode/__init__.py

```diff
@@ -80,6 +82,21 @@ def align_scans(self):
             for f in self.old.files:
                 f.original_path = f.path
 
+    def similarity(self):
+        for delta in self.deltas:
+            if delta.new_file == None or delta.old_file == None:
+                continue
+            new_fingerprint = delta.new_file.fingerprint
+            old_fingerprint = delta.old_file.fingerprint
+            if new_fingerprint == None or old_fingerprint == None:
+                continue
+            new_fingerprint = utils.bitarray_from_hex(delta.new_file.fingerprint)
+            old_fingerprint = utils.bitarray_from_hex(delta.old_file.fingerprint)
+            hamming_distance = utils.hamming_distance(new_fingerprint, old_fingerprint)
+            delta.score += hamming_distance
+            if hamming_distance <= SIMILARITY_THRESHOLD:
```

Comment:
While this is a good start, we need to two additions:
1) We want to add some extra value to the score: `Delta.score += something` The thinking here is that files that are more dissimilar should have a greater score value that those that are more similar.
2) This score addition need to be stepped, meaning we want to check against different thresholds and increase the score accordingly. Something like `LOW_THRESHOLD`, `MED_THESHOLD`, and `HIGH_THRESHOLD`